docs/steps.md

```diff
@@ -0,0 +1,100 @@
+### Node Setup
+Here we assume a typical Ubuntu 22.04 LTS x86_64 setup. If you are using a different OS you may need to make some adjustments.
+For more information about the compute node requirement see [here](hosting.md)
+
+Make sure `jq`, `git`, `curl`, and `make` are installed. 
+
+```bash
+sudo apt update
+sudo apt install -y jq curl git 
+jq --version
+```
+
+Now clone the network github repository
+```bash
+git clone https://github.com/zeta-chain/network-athens3.git && cd network-athens3
+```
+
+Get two executable binaries (`zetacored` and `zetaclientd`)
+- [ ] Add instructions here
+- [ ] Copy the binaries to `/usr/bin/` folder
+
+Check out a specific branch. Branch name will be provided by the coordinator
+```bash
+git checkout <Branch-Name>
+```
+
+Give execute permissions to the scripts and run the node setup script
+```bash
+chmod +x ./scripts/*.sh
+```
+
+```bash
+./scripts/node-setup.sh
+```
+
+After the `node-setup.sh` script generates the keys and necessary
+files, create a branch, commit, and raise a PR to submit the files
+to zetachain coordinator:
+
+  - Use `Branch-Name-<YourName>` as the branch name for the new branch.
+  - The pr should contain only `os_info.json` and `gentx-XX.json`. Use `git status` to check
+  - If there is a file `network_files/config/genesis.json` do not commit it.This file can be deleted ,it is not required.The genesis being used will be provided by the coordinator.
+
+NOTE : A backup us created for the existing zetacored folder under `~/.zetacored_old/zetacored-<timestamp>`.You can copy back keys etc if needed .
+
+  
+
+### Start Node Step
+
+**Wait for final genesis to be provided by the Coordinator of the ceremony before starting the following process**
+
+
+Edit config file (~/.zetacored/config/config.toml) to
```

Comment:
@kingpinXD i think we need to git pull here get the latest genesis.json file right?

Reply:
^^ @kingpinXD